### PR TITLE
GCP service account

### PR DESCRIPTION
**Pull request by azure562**
- Clarify the level of SA and role
- Update screenshots

See CMP-4834

### CODE REVIEW

File: website/docs/google-cloud/connect-google-cloud-service-account.mdx [hunk 1]
```diff
@@ -5,116 +5,123 @@ description: Connect CMP to your Google Cloud Organization to unlock additional
...
+   A slide-out will appear on the right-hand side of the screen containing the Google SDK ([gcloud](https://cloud.google.com/sdk)) commands you will need to run in order to create the service account and set it up with the correct role.
 
-  - The project must be connected to a [Google Cloud Billing account](https://cloud.google.com/billing/docs/concepts#billing_account).
+   :::info
 
-  - The project should be one that you are planning to keep long-term and that will only be accessible by only trusted people.
+   Before executing the `gcloud` commands, ensure that:
 
-- Your Google Cloud user account has the [Organization Role Administrator role](https://cloud.google.com/iam/docs/understanding-custom-roles#organization_role_administrator_role) or, alternatively, the [Role Administrator role](https://cloud.google.com/iam/docs/understanding-custom-roles#role_administrator_role) for the project you have selected.
+   - You have selected an appropriate Google Cloud project to create your service account.
```
Comment (mtagda) [inline website/docs/google-cloud/connect-google-cloud-service-account.mdx:50]: @azure562 shouldn't we add a note here that the project must belong to the org? cc @taltultc

Comment (taltultc) [reply website/docs/google-cloud/connect-google-cloud-service-account.mdx:50]: Yes we should

**Review by mtagda — APPROVED** [review]

File: website/docs/google-cloud/connect-google-cloud-service-account.mdx [hunk 2]
```diff
@@ -5,116 +5,125 @@ description: Connect CMP to your Google Cloud Organization to unlock additional
...
-To add a new service account, click on the "**+**" icon in the top-right corner of the _Configured Accounts_ widget on the left-hand side of the screen.
+To add a new service account, select the plus icon (**+**) in the upper-right corner in the **Configured accounts** widget on the left-hand side of the screen.
 
-Once clicked, notice that all of the Features will revert back to "Not Configured" since you are setting up a new service account.
+Once clicked, all of the Features will revert back to "Not Configured" because you are setting up a new service account.
 
 As you did with your initial service account, select the features you'd like to grant permissions for this service account, and run the `gcloud` commands provided.
 
-Check out the bite-sized video below for a closer look at connecting your Google Cloud Organization to CMP.
+## Watch a demo
```
Comment (norosa) [inline website/docs/google-cloud/connect-google-cloud-service-account.mdx:126]: Is this one of the instances where we can comment out the video to remove it or do you think it has info in there we should preserve for now?

**Review by norosa — APPROVED** [review]

File: website/docs/google-cloud/connect-google-cloud-service-account.mdx [hunk 3]
```diff
@@ -5,116 +5,125 @@ description: Connect CMP to your Google Cloud Organization to unlock additional
...
-The `gcloud` command achieves the following: the service account is created in one of the projects in your organization, and then the permissions for this service account are given on the organization level.
+   :::
 
-:::
+   :::tip
+
+   The `gcloud` commands achieve the following:
+
+   - Creates the service account in one of the projects in your organization.
+   - Grants the permissions for the service account on the organization level.
```
[the remark this answers is not in the document]
Comment (azure562) [reply website/docs/google-cloud/connect-google-cloud-service-account.mdx:75]: Google's style guide suggests using a period except for "single-word items, items entirely in code font, items with no verbs." See https://developers.google.com/style/periods?hl=en#bulleted-lists

**Pull request merged**